### DISCUSSION TITLE
One vim mode for all open tabs

### DISCUSSION THREAD

**Discussion by viczem** [JetBrains/ideavim#410] [Q&A]
For example, Tab A has insert mode, after switch to Tab B by <kbd>Ctrl</kbd> + <kbd>Tab</kbd>, vim mode of the Tab B may be different. Is it possible to configure Ideavim to keep one mode for all tabs?

(e.g. changing Tab A to normal mode, will set normal mode for all others opened tabs)

**Comment by AlexPl292** [comment]
Hi, please follow this issue: https://youtrack.jetbrains.com/issue/VIM-2505
